README.md

```diff
@@ -38,4 +38,11 @@ Select from two editions:
 [Feature Comparison](Content/FeatureComparison.md) | [System requirements](https://github.com/sandboxie-plus/Sandboxie#sandboxie) | [Changelog](https://github.com/sandboxie-plus/Sandboxie/blob/master/CHANGELOG.md) | [Download the latest release](https://github.com/sandboxie-plus/Sandboxie/releases/latest)
 
 ## Contribute
+
 If you have development, testing or translation skills, then feel free to check out our [Contribution guidelines](https://github.com/sandboxie-plus/Sandboxie/blob/master/CONTRIBUTING.md).
+
+Specifically, for this repo:
+* Install `pre-commit` (`pip install pre-commit`; see more ways in https://pre-commit.com/#installation), and
+* At the root of the repository, run `pre-commit install`
+
+This ensures that your commits are formatted according to project standards.
```

Comment:
Are these instructions still valid with the latest code changes?

Reply:
Yes - how we integrate CI (the "absolute gatekeeper") does not change how preferably the contributors should set up their environment